builder/builder-next/exporter/export.go

```diff
@@ -196,14 +198,13 @@ func (e *imageExporterInstance) Export(ctx context.Context, inp exporter.Source,
 	}
 	_ = configDone(nil)
 
-	if e.opt.ReferenceStore != nil {
-		targetNames := strings.Split(e.targetName, ",")
-		for _, targetName := range targetNames {
-			tagDone := oneOffProgress(ctx, "naming to "+targetName)
+	if e.opt.ReferenceStore != nil && e.targetName != "" {
+		for _, targetName := range strings.Split(e.targetName, ",") {
 			tref, err := distref.ParseNormalizedNamed(targetName)
```

Comment:
Do we need a check inside the loop to check for;

- `targetName` being empty
- `targetName` being a whitespace-only string?


Thinking of cases where (idk) something like `"   , , , "` would ever find it's way here

Reply:
So something like;

```go
targetName = strings.TrimSpace(targetName)
if targetName == "" {
	continue
}
```